src/main/appengine/index.yaml

```diff
@@ -93,3 +93,15 @@ indexes:
     name: targetUser
   - direction: asc
     name: endTime
+- kind: Course
+  properties:
+  - direction: asc
+    name: isMigrated
```

Comment:
Ok I checked on this, one index would be sufficient if we decide are migrating courses in one direction. But since we're going to initially migrate courses oldest to newest, then newest to oldest (after we're confident our script works) we would need both